lib/extends/controllers/decidim/devise/sessions_controller_extends.rb

```diff
@@ -1,24 +1,28 @@
 # frozen_string_literal: true
 
 module SessionControllerExtends
-  def after_sign_in_path_for(user)
-    if user.present? && user.blocked?
-      check_user_block_status(user)
-    elsif first_login_and_not_authorized?(user) && !user.admin? && !pending_redirect?(user) && !skip_authorization_handler?
-      decidim_verifications.first_login_authorizations_path
-    else
-      super
+  extend ActiveSupport::Concern
+
+  included do
+    def after_sign_in_path_for(user)
+      if user.present? && user.blocked?
+        check_user_block_status(user)
+      elsif !skip_fist_login_authorization? && (first_login_and_not_authorized?(user) && !user.admin? && !pending_redirect?(user))
+        decidim_verifications.first_login_authorizations_path
+      else
+        super
+      end
     end
-  end
 
-  private
+    private
 
-  # Skip authorization handler by default
-  def skip_authorization_handler?
-    ENV["SKIP_FIRST_LOGIN_AUTHORIZATION"] ? ActiveRecord::Type::Boolean.new.cast(ENV["SKIP_FIRST_LOGIN_AUTHORIZATION"]) : true
+    # Skip authorization handler by default
+    def skip_fist_login_authorization?
```

Comment:
Typo here

Reply:
Could you explain what is a typo?

Reply:
You wrote "fist" instead of "first"

Reply:
Thanks you, for this kind of issue could you add a suggestion next time? 
<img width="473" alt="Capture d’écran 2022-11-25 à 18 34 30" src="https://user-images.githubusercontent.com/20232956/204033919-0f7c8cad-f160-4545-8d4a-a17485096a2f.png">